wiki/fr/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: wikioverview
+title: "Vue d'ensemble"
+lang: "fr"
+permalink: "/wiki/"
+---
+# Page overview
```

Comment:
I think this also needs to be translated into french.

Reply:
Done!